VitalsSimplification/AbnormalityChecker.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace vitals_simplification
+{
+    class AbnormalityChecker

```

Comment:
Looks like AbnormalityChecker is not used?

Reply:
I created AbnormalityChecker class to check whether a particular vital is low or high and print accordingly as mentioned in the tasks. And also I didn't get how to verify the output results(test cases) for AbnormalityChecker.

Reply:
Was it clear after our last classroom on unit testing?